unicodetools/data/emoji/13.1/ReadMe-ucd-emoji.txt

```diff
@@ -0,0 +1,16 @@
+This directory contains DRAFT data files for Unicode, Version 13.0
```

Comment:
Shouldn't this and the other files below have Version 13.1?

Reply:
I should explain more. Just to get everything up and running, I copy the old files into the new directory. Once everything is clean, then I rebuild the data files using the union of candidateData plus these files. Then swap those new files in. So this is temporary.

Reply:
So if you could skip the data/emoji/13.1 files for now...